File: 2026-09/src/day_13/prms_120956/Solution.java
package day_13.prms_120956;

class Solution {
    public int solution(String[] babbling) {
        int cnt = 0;
        for (String s : babbling) {
            String res = s
                    .replace("aya", " ")
                    .replace("ye", " ")
                    .replace("woo", " ")
                    .replace("ma", " ");

            if (res.isBlank()) cnt++;
        }

        return cnt;
    }
}

File: 2026-09/src/day_13/prms_120956/Solution.cpp
#include <bits/stdc++.h>
using namespace std;

int solution(vector<string> babbling) {
    int cnt = 0;
    for (string& s : babbling) {
        for (string p : {"aya", "ye", "woo", "ma"}) {
            int pos = s.find(p);
            if (pos != -1) s.replace(pos, p.size(), " ");
        }

        bool ok = true;
        for (char c : s) {
            if (!isspace(c)) ok = false;
        }
        if (ok) cnt++;
    }

    return cnt;
}
